### PR TITLE
Document element commands

### PR DESCRIPTION
**Pull request by dpgraham**
* Did 6 more endpoints
* Made some fixes in some other YML
* Updated the template.md so the description can be excluded (because often time a short_description is sufficient)
* Made it so that the HTTP response section in template.md isn't always a table
* Committed generated markdown files

### CODE REVIEW

File: commands-yml/commands/element/actions/send-keys.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+name: Send Keys
+short_description: Send a sequence of key strokes to an element
+description:
+  >
+    Any UTF-8 character may be specified, however, if the server does not support native key events, it should simulate key strokes for a standard US keyboard layout. The Unicode Private Use Area code points, 0xE000-0xF8FF, are used to represent pressable, non-text keys (see table below).
```
Comment (imurchie) [inline commands-yml/commands/element/actions/send-keys.yml]: Could also link to the unicode documentation for how to allow it in Android.

File: commands-yml/commands/element/attributes/attribute.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+---
+name: Get Element Attribute
+short_description: Get the value of an element's attribute
```
Comment (imurchie) [inline commands-yml/commands/element/attributes/attribute.yml]: Do we have any information about what attributes are available?

File: commands-yml/commands/element/find-elements.yml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+short_description: Search for multiple elements
+description:
+  >
+     Get a list of elements that match the [locator selector](/docs/en/about-appium/getting-started).
+selector_strategies:
+  # TODO Document the rest
+  - name: Accessibility ID
+    description: Read a unique identifier for a UI element. For XCUITest it is the element's `resource-id` attribute. For Android it is the element's `content-desc` attribute.
+  - name: Class name
+    description: "For IOS it is the full name of the XCUI element and begins with XCUI. For Android it is the full name of the UIAutomator2 class (e.g.: android.widget.TextView)"
```
Comment (imurchie) [inline commands-yml/commands/element/find-elements.yml]: Why is this in quotes? Also, the XCUITest part should start with `XCUIElementType`.

File: commands-yml/commands/element/other/active.yml [hunk 4]
```diff
@@ -0,0 +1,74 @@
+---
+name: Get Active Element
+short_description: Gets the active element of the current session
+description:
+  >
+    Get the element on the page that currently has focus.
```
Comment (imurchie) [inline commands-yml/commands/element/other/active.yml]: This doesn't seem to add much.

File: commands-yml/commands/element/other/submit.yml [hunk 5]
```diff
@@ -0,0 +1,79 @@
+---
+name: Submit Form
+short_description: Submit a FORM element
+description:
+  >
+     The submit command may also be applied to any element that is a descendant of a FORM element.
+
+     This feature is only supported from within WebView context.
```
Comment (imurchie) [inline commands-yml/commands/element/other/submit.yml]: Need an article here: `within a WebView context`.

**Review by imurchie — COMMENTED** [review]
A few things. But awesome!

**Comment by imurchie** [comment]
Is this meant to supersede #9281?

File: commands-yml/commands/element/other/active.yml (hunk 4 above)
Comment (dpgraham) [reply commands-yml/commands/element/other/active.yml]: Yeah, that was before when description was mandatory. I'll take it out.

**Comment by dpgraham** [comment]
@imurchie Yeah, it's ahead of the tip of #9281. 9281 had changes that this PR needed.

File: commands-yml/commands/element/other/submit.yml (hunk 5 above)
Comment (imurchie) [reply commands-yml/commands/element/other/submit.yml]: Sorry. I meant a [grammatical article](https://en.wikipedia.org/wiki/Article_(grammar)) :)

Comment (imurchie) [reply commands-yml/commands/element/other/submit.yml]: I'm working on re-working the context stuff and all the related docs (hybrid, web, iwdp, etc)

File: commands-yml/commands/element/attributes/attribute.yml (hunk 2 above)
Comment (dpgraham) [reply commands-yml/commands/element/attributes/attribute.yml]: I'm not sure. I couldn't find a document that listed all of that.

**Comment by dpgraham** [comment]
Fixes made

**Review by imurchie — APPROVED** [review]

**Pull request merged**